src/components/vr_cooperation/src/on_default_service_chosen.cc

```diff
@@ -56,8 +57,10 @@ void OnDefaultServiceChosen::Execute(
   if (value.isMember(json_keys::kParams)
       && value[json_keys::kParams].isMember(json_keys::kAppId)) {
     uint32_t app_id = value[json_keys::kParams][json_keys::kAppId].asUInt();
-    LOG4CXX_DEBUG(logger_, "hmi_app_id: " << app_id);
+    LOG4CXX_DEBUG(logger_, "Default app id for VR service: " << app_id);
     parent_->set_stored_app_id(app_id);
+  } else {
+    LOG4CXX_ERROR(logger_, "No app id found in HMI message");
```

Comment:
From protocol: 

> <description>appID which service is choosen by user to be default. In case of no appID defined, default service has been reset and HU-native service is uses by default</description>

I don't think that is error, maybe info and I don't see logical to reset default service to HU-native